StageGIM/Student Management/Student Management/Assignment-2/Main.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using StudentRecordManagementSystem;
+
+namespace StudentRecordManagementSystem
+{
+    class MainWindow
+    {
+        static void Main(string[] args)
+        {
+            Student kid = new Student();
+            HelpDeskSystem help = new HelpDeskSystem();
+            StudentManagement man = new StudentManagement();
+            bool Running = true;
+
+            while (Running) 
+            {
+                Console.WriteLine("Choose an option:");
+                Console.WriteLine("1. Add Student");
+                Console.WriteLine("2. Add Grade");
+                Console.WriteLine("3. Display Average Grade");
+                Console.WriteLine("4. Add student to HelpDesk Queue");
+                Console.WriteLine("5. Look Up Student");
+                Console.WriteLine("6. Remove Student");
+                Console.WriteLine("7. Update Student");
+                Console.WriteLine("8. Find and Display Student");
+                Console.WriteLine("9. Exit");
+
+                string Input =Console.ReadLine();
+
+                switch (Input)
```

Comment:
mention the cases in comments above each case below

Reply:
Did I do it like you meant what I needed to do?